lib/auth/grpcserver.go

```diff
@@ -5426,6 +5426,7 @@ func NewGRPCServer(cfg GRPCServerConfig) (*GRPCServer, error) {
 
 	staticHostUserServer, err := userprovisioningv1.NewService(userprovisioningv1.ServiceConfig{
```

Comment:
Unrelated to your changes, but would you mind fixing this import?
```suggestion
	staticHostUserServer, err := userprovisioningv2.NewService(userprovisioningv1.ServiceConfig{
```